libraries/abi_generator/abi_generator.cpp

```diff
@@ -595,16 +649,24 @@ string abi_generator::add_type(const clang::QualType& tqt, size_t recursion_dept
     is_type_def = true;
   }
 
+  bool fixed_array = is_fixed_array(qt);
```

Comment:
make const to make it obvious that it isn't set anywhere else

Reply:
ok